synfig-studio/src/gui/duckmatic.cpp

```diff
@@ -515,11 +515,10 @@ Duckmatic::update_ducks()
 							{
 								// Create a new BLinePoint
 								BLinePoint nbp;
-								// Terporary set the flags for the new BLinePoint to all split
+								// Temporarily set the flags for the new BLinePoint to all split
 								nbp.set_split_tangent_both(true);
 								// Now we can set the tangents. Tangent2 won't be modified by tangent1
-								nbp.set_tangent1(c1->get_point());
-								nbp.set_tangent2(bp.get_tangent1());
+								nbp.set_tangents(c1->get_point(), c1->get_point());
 								// Now update the flags
 								nbp.set_split_tangent_radius(bp.get_split_tangent_radius());
 								nbp.set_split_tangent_angle(bp.get_split_tangent_angle());
```

Comment:
Part 2: restoring the flags here... 
So maybe both temporary-set, restore-set lines should be deleted ;)